jujugui/static/gui/src/app/widgets/autodeploy-extension.js

```diff
@@ -75,7 +75,6 @@ YUI.add('autodeploy-extension', function(Y) {
         parentId: parentId,
         constraints: constraints || {}
       }], callback, {modelId: machine.id});
-      db.fire('update');
```

Comment:
This is a change I had previously made to make the UI update on machine creation, this has now been replaced with the listener in app.js.